packages/website/components/account/filesManager/filesManager.js

```diff
@@ -103,12 +104,10 @@ const FilesManager = ({ className, content, onFileUpload }) => {
     }
   }, [query, currentTab, pinned, replace]);
 
-  // Initial fetch on component load
+  // Re-fetch uploads when props change
   useEffect(() => {
-    if (!fetchDate && !isFetchingUploads) {
-      getUploads();
-    }
-  }, [fetchDate, getUploads, isFetchingUploads]);
+    getUploads({ size: itemsPerPage, page, sortBy, sortOrder });
```

Comment:
Might be good to reset the `selectedFiles` here too, so that hidden selections aren't stored in that state.